client/my-sites/plans-features-main/hooks/experiments/use-longer-plan-term-default-experiment.ts

```diff
@@ -1,55 +0,0 @@
-import { ONBOARDING_FLOW } from '@automattic/onboarding';
-import { useExperiment } from 'calypso/lib/explat';
-
-const useTermFromExperimentVariant = (
-	experimentVariant: string | null | undefined
-): string | null => {
-	switch ( experimentVariant ) {
-		case 'default_to_three_year_plans':
-			return '3yearly';
-		case 'default_to_two_year_plans':
-			return '2yearly';
-		default:
-			return null;
-	}
-};
-
-/**
- * Returns:
- * - the term as yearly, 2yearly, 3yearly (etc.) to be used as default in the plans
- * page based on the experiment variant
- * - OR null/undefined if control variant or if the experiment is still loading (Subject to change)
- *
- * This hook although used for the experiment, it can be refactored in the end to
```

Comment:
`Non-blocking:`

It's been a while so I don't have the clearest memory, but there was some complexity around calculating the default plan term used for the plans grid. This hook was meant to be repurposed to that end after the conclusion of the emphasize longer term plan savings experiment.

It might be worth double-checking if this is still the case.

Reply:
I'll create an issue for this before I sign off.

Reply:
https://github.com/Automattic/martech/issues/3756